stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.7.1
+version: 1.8.0
```

Comment:
from my last PR it was concluded that this change might still be a minor version bump.